data_ingestion.py

```diff
@@ -25,23 +25,27 @@ def get_emission_data(self):
 		response = requests.get(self.emission_api)
 		# Print the status code of the response.
 		#print(response.status_code)
-
 		emission_data = json.dumps(response.json())
-
-		df_emission_data = pd.read_json(emission_data)
-		def Clean_names(City_name):
-			"""To remove the state name from city column
-			using regex. everything after , in city column is removed
-			"""
-			if re.search(r'\,.*', City_name): 
-				pos = re.search(r'\,.*', City_name).start() 
-				return City_name[:pos] 
-			else:
-				return City_name
-		df_emission_data['city'] = df_emission_data['city'].apply(Clean_names) 
+		df_emission_data = pd.read_json(emission_data) 
 		print(df_emission_data.head())
 		return df_emission_data
 
+	def Clean_names(self):
+		"""To remove the state name from city column
+		Parameters: City_name - to look upon city column
+		Returns: A dataframe where state short form removed from city column
+		"""
+		k=[]
+		df_emission_data=self.get_emission_data()
+		for x in df_emission_data.city:
```

Comment:
Use lamda function and apply method instead of looping when using pandas dataFrame